### DISCUSSION TITLE
PROBLEM 2 FOR HW 1: I am unsure where to begin I need a push in the right direction if possible.

### DISCUSSION THREAD

**Discussion by MohamedHaithamAhmed** [cooperrc/computational-mechanics#220] [General]
2. Use the variables given below, `str1` and `str2`, and check the following 

    a. `str1<str2`
    
    b. `str1==str2`
    
    c. `str1>str2`
    
    d. How could you force (b) to be true? [Hint](https://docs.python.org/3/library/stdtypes.html?highlight=str.lower#str.lower) or [Hint](https://docs.python.org/3/library/stdtypes.html?highlight=str.lower#str.upper)

**Comment by mim20017** [comment]
You're just looking for it to return 'true' or 'false', the first one for example I just did print(str1<str2) and it told me it was true. D is a little tricky but with the help of the first hint provided it didn't take long to figure it out.

**Reply by cooperrc** [reply]
Nice! e.g. 

```python
>>> str1 = 'Python'
>>> str2 = 'python'
>>> str1 < str2
True
```
